docs/Ingress.md

```diff
@@ -383,6 +383,8 @@ spec:
 
 If you only have one indexer cluster that you would like to use as the destination for all S2S traffic, you can optionally replace `splunk.example.com` in the above examples with the wildcard `*`. When you use this wildcard, you do not have to set the `tlsHostname` parameter in `outputs.conf` on your forwarders.
 
+Additionally, it is necessary to push an inputs.conf to both configure the relevant TLS settings and open port 9998 for receiving on the indexers as these configurations are not default. This can be achieved by installing an app containing a custom inputs.conf to the indexers through the Operator's [app management capabilities](https://splunk.github.io/splunk-operator/Examples.html#installing-splunk-apps).
```

Comment:
4. Deploy an app to the standalone instance with the inputs.conf settings needed to open port 9998 and configure the relevant TLS settings. For details on app management using the Splunk Operator, see [Using Apps for Splunk Configuration](Examples.md#using-apps-for-splunk-configuration).

Reply:
Fixed